packages/react/src/components/Storage/FileUploader/FileUploader.tsx

```diff
@@ -352,23 +350,51 @@ export function FileUploader({
     setAutoLoad(false);
   }, [shouldAutoProceed, onFileClick, autoLoad, hasMaxFilesError]);
 
+  const hiddenInput = React.useRef<HTMLInputElement>();
+
+  const accept = acceptedFileTypes?.join();
+  const uploadButton = useMemo(
+    () => (
+      <>
+        <UploadButton
+          onClick={() => {
+            hiddenInput.current.click();
+            hiddenInput.current.value = null;
+          }}
+          className={ComponentClassNames.FileUploaderDropZoneButton}
+        />
+        <VisuallyHidden>
+          <input
+            type="file"
+            tabIndex={-1}
+            ref={hiddenInput}
+            onChange={onFileChange}
+            multiple={hasMultipleFiles}
+            accept={accept}
+          />
+        </VisuallyHidden>
+      </>
+    ),
+    [accept, hasMultipleFiles, onFileChange, UploadButton]
+  );
+
   if (showPreviewer) {
     return (
       <Previewer
-        acceptedFileTypes={acceptedFileTypes}
+        dropZone={
+          <UploadDropZone {...dropZoneProps} inDropZone={inDropZone}>
+            <Text className={ComponentClassNames.FileUploaderDropZoneText}>
+              {translate('Drop files here or')}
+            </Text>
```

Comment:
This could probably just go in the DropZone itself maybe

Reply:
Done in the main `fileuploader/main` code